### PR TITLE
EKF add estimator namespace and cleanup whitespace

### PR DESCRIPTION
**Pull request by dagar**
Namespace and minor whitespace changes from https://github.com/PX4/ecl/pull/613.

### CODE REVIEW

File: EKF/covariance.cpp [hunk 1]
```diff
@@ -77,15 +80,15 @@ void Ekf::initialiseCovariance()
...
 
 	} else if (_control_status.flags.gps_hgt) {
-		float lower_limit = fmaxf(_params.gps_pos_noise, 0.01f);
+		float lower_limit = fmaxf(_params.gps_pos_noise, 0.01);
 		float upper_limit = fmaxf(_params.pos_noaid_noise, lower_limit);
 		P[9][9] = sq(1.5f * math::constrain(_gps_sample_delayed.vacc, lower_limit, upper_limit));
 
 	} else {
-		P[9][9] = sq(fmaxf(_params.baro_noise, 0.01f));
+		P[9][9] = sq(fmaxf(_params.baro_noise, 0.01));
```
[the remark this answers is not in the document]
Comment (dagar) [reply EKF/covariance.cpp]: Yes, that's probably an accidental carryover from experimenting in https://github.com/PX4/ecl/pull/613.

[the remark this answers is not in the document]
Comment (dagar) [reply EKF/covariance.cpp]: Fixed.

**Comment by dagar** [comment]
Looks like the python testing needs to be updated to work with the namespace.

**Comment by priseborough** [comment]
@dagar for dummies like me can you explain the benefit of the namespace change? TXS :)

**Comment by jkflying** [comment]
> can you explain the benefit of the namespace change? TXS :)

The idea is that if someone else has a free-standing method or a class with the same name, it won't collide. This would let us add an ecl::Filter as well as an xyz::Filter in the same compiled binary for example.

**Pull request closed without merging**